backend/src/main/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhir.java

```diff
@@ -219,8 +220,8 @@ private Bundle convertRowToFhirBundle(TestResultRow row, UUID orgId) {
                 .dob(LocalDate.parse(row.getPatientDob().getValue(), DATE_TIME_FORMATTER))
                 .address(patientAddr)
                 .country(DEFAULT_COUNTRY)
-                .race(row.getPatientRace().getValue())
-                .ethnicity(row.getPatientEthnicity().getValue())
+                .race(getRaceLiteral(row.getPatientRace().getValue()))
+                .ethnicity(getEthnicityLiteral(row.getPatientEthnicity().getValue()))
```

Comment:
Does it make sense to use a different map here than the one we're using in the fhir converter?

The race map in the fhir converter is throwing me for a loop by having values that go both ways.
```
    raceMap.put("native", NATIVE_CODE);
    raceMap.put(NATIVE_CODE, "native");
```

If we're going to map the bulk upload accepted values to the single entry expected equivalent only to map it back to the loinc code (which could have been the bulk upload starting value) I think it makes sense for that first map to be in the bulk upload code and be explicit in what its doing.